resources/knative-serving/README.md

```diff
@@ -12,4 +12,4 @@ Kyma-specific changes:
  * The `config-domain` is made configurable by specifying the `.Values.global.domainName` as the helm template.
  * The `knative-serving` Namespace is no longer created. This happens during the installation process.
  * The image versions are changed to use the release tag.
- * Knative Serving uses the Kyma Istio Ingress gateway.
+ * knative-ingress-gateway has been updated
```

Comment:
we might want to point out that the knative-ingress-gateway is now basically a copy of the kyma-gateway, just in a different namespace